### DISCUSSION TITLE
Adding support for StoryLocations

### DISCUSSION THREAD

**Discussion by mycroftcanner** [sbertix/Swiftagram#223] [Ideas]
Hey all,

I'm adding support for some properties I was using in `SwiftyInsta` to `Swiftagram`, one of them being `StoryLocations`

In `SwiftyInsta` we did:

```
extension Media { 
/// The `storyLocations` value.
    public var locations: [StoryLocation] {
        return rawResponse.storyLocations
            .array?
            .compactMap(StoryLocation.init) ?? []
    }
}
```

Is this the equivalent implementation in `Swiftagram`?:

```
extension Media {
  /// The `storyLocations` value.
  public var storyLocations: [Location]? {
    self["storyLocations"]
      .array()?
      .map(Location.init)
  }
}
```

Cheers

**Comment by sbertix** [comment]
Hey @mycroftcanner, 

I have no way to check right now if `Location` and the object in `storyLocations` have the same definition, and follow the same model. 
However, that might be the case. 
Other than that, yes, the two snippets are equivalent. 

I might be able to check `Location` and add it tomorrow though. I've already written models for stories polls and music, so I'm counting on adding them all in time for next release 💪
